recipes/qhull/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qhull" %}
+{% set version = "2015.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/qhull/qhull.git
```

Comment:
Instead of github, use the url from http://www.qhull.org/download/qhull-2015-src-7.2.0.tgz for all 3 platforms